### DISCUSSION TITLE
Questions about free energy in the official case DerivativeMultiPhaseMaterial.i

### DISCUSSION THREAD

**Discussion by tmewhy** [idaholab/moose#17816] [Q&A Modules: Phase field] [answered]
Hi guys
I am studying the official case **DerivativeMultiPhaseMaterial.i**. 

> (modules/phase_field/examples/multiphase/DerivativeMultiPhaseMaterial.i)

In this case, the free energy is set to **F1=(c-1)^2, F2=(c-0.5) ^2, F3=c ^2**, the initial conditions set the concentration c to smoothly change from 1 to 0. According to my own understanding, according to the principle of minimizing free energy, c=1, c=0.5, c=0, they should all be balanced phases, but the simulation result is that **only c=0.5, c=0 are left**.  I’m very confused about this. Is my understanding of free energy not clear enough, or what setting was used in the case to cause c=1 not to be a minimum free energy phase.

**Comment by simopier** [comment, marked as answer]
Hi, 

as you noticed, the phase with `c=1` disappears in that simulation, despite `c=1` being a minimum for the bulk free energy defined as `Fb = h1*F1+h2*F2+h3*F3`. This is because the total free energy also includes other sources of energy.

As you can see in the input file, it includes:
* an energy barrier defined by the `MultiBarrierFunctionMaterial` block and included in the energy by the `free_energy` block (g(&eta;))
* the cross energies for the order parameters defined in the `cross_terms` block (the equivalent of &kappa;/2 |&nabla; &eta;| <sup>2</sup> for a multi-phase system)
* an energy term related to the gradient of the concentration introduced in `local_free_energy` (&kappa;/2 |&nabla; c| <sup>2</sup>)

The microstructure evolves to reduce the integral of the **total** free energy, not only the bulk free energy contribution. By loosing the center phase with `c=1`, the system reduces the interfacial free energy due to the concentration and order parameter gradients. 

I hope that helps!

**Reply by tmewhy** [reply]
I have fully understood this question through your answer, thank you very much for your help!

**Reply by simopier** [reply]
You are welcome!
